packages/chart/src/CdcChartComponent.tsx

```diff
@@ -51,11 +49,11 @@ import useDataVizClasses from '@cdc/core/helpers/useDataVizClasses'
 import numberFromString from '@cdc/core/helpers/numberFromString'
 import getViewport from '@cdc/core/helpers/getViewport'
 import { DataTransform } from '@cdc/core/helpers/DataTransform'
-import cacheBustingString from '@cdc/core/helpers/cacheBustingString'
```

Comment:
This was used when the response was always loaded from the url to force cache to request a the new config if fifteen minutes had passed. Make sure this stays in place on the fetch.

Reply:
Fixed!